site/en/blog/autoplay/index.md

```diff
@@ -0,0 +1,290 @@
+permission, it can delegate that permission to cross-origin iframes with the
+[permissions policy for autoplay]. Note that autoplay is allowed by default on
+same-origin iframes.
+
+```html
+&lt;!-- Autoplay is allowed. -->
+&lt;iframe src="https://cross-origin.com/myvideo.html" allow="autoplay">
+
+&lt;!-- Autoplay and Fullscreen are allowed. -->
+&lt;iframe src="https://cross-origin.com/myvideo.html" allow="autoplay; fullscreen">
+```
+
+When the permissions policy for autoplay is disabled, calls to `play()` without a
+user gesture will reject the promise with a `NotAllowedError` DOMException. And
+the autoplay attribute will also be ignored.
+
+{% Aside 'warning' %} Older articles incorrectly recommend using the attribute
+`gesture=media` which is not supported.{% endAside %}
+
+### Examples
+
+**Example 1:** Every time a user visits `VideoSubscriptionSite.com` on their
+laptop they watch a TV show or a movie. As their media engagement score is
+high, autoplay is allowed.
+
+**Example 2:** `GlobalNewsSite.com` has both text and video content.
+Most users go to the site for text content and watch videos only occasionally.
+Users' media engagement score is low, so autoplay wouldn't be allowed if a user
+navigates directly from a social media page or search.
+
+**Example 3:** `LocalNewsSite.com` has both text and video content.
+Most people enter the site through the homepage and then click on the news
+articles. Autoplay on the news article pages would be allowed because of user
+interaction with the domain. However, care should be taken to make sure users
+aren't surprised by autoplaying content.
+
+**Example 4:** `MyMovieReviewBlog.com` embeds an iframe with a movie trailer to
+go with a review. Users interacted with the domain to get to the blog, so
+autoplay is allowed. However, the blog needs to explicitly delegate that
+privilege to the iframe in order for the content to autoplay.
+
+### Chrome enterprise policies
+
+It is possible to change the autoplay behavior with Chrome enterprise policies
+for use cases such as kiosks or unattended systems. Check out the [Policy
+List](policy-list) help page to learn how to set the autoplay related enterprise
+policies:
+
+- The `[AutoplayAllowed](autoplayallowed)` policy controls whether autoplay is allowed or not.
+- The `[AutoplayAllowlist](autoplayallowlist)` policy allows you to specify an allowlist of URL
+  patterns where autoplay will always be enabled.
+
+## Best practices for web developers
+
+### Audio/Video elements
+
+Here's the one thing to remember: Don't ever assume a video will play, and
+don't show a pause button when the video is not actually playing. It is so
+important that I'm going to write it one more time below for those who simply
+skim through that post.
+
+{% Aside 'gotchas' %}Don't assume a video will play, and don't show a pause
+button when the video is not actually playing.{% endAside %}
+
+You should always look at the [Promise](promise) returned by the play function
```

Comment:
Promise link does not work. Use https://web.dev/promises/

Reply:
I think it's fixed.